internal/binding/binding.go

```diff
@@ -128,6 +128,10 @@ type Binding interface {
 	// DialIxNetwork creates a client connection to the specified ATE's IxNetwork endpoint.
 	DialIxNetwork(ctx context.Context, ate *reservation.ATE) (*IxNetwork, error)
 
+	// DialGRIBI creates a client connection to the specified DUT's gRIBI endpoint.
+	// Implementations must append transport security options necessary to reach the server.
+	DialGRIBI(ctx context.Context, dut *reservation.DUT, opts ...grpc.DialOption) (*GRIBIClient, error)
```

Comment:
I think at the binding level, we should just return the gribi.proto's GRIBIClient, not the fluent client.

https://pkg.go.dev/github.com/openconfig/gribi@v0.1.0/proto/service#GRIBIClient

This is consistent with gNMI, P4RT, etc. Also update the the RawAPIs to return a GRIBIClient proto stub?

Reply:
Thanks! agree. I plan to separate the feature dependent code from the binding and in that case it makes sense to return GRIBIClient rather than the fluent.